eclipse-theia/theia
# dep: remove `font-awesome-webpack`

#### What it does


Fixes: #10400 

The pull-request removes `font-awesome-webpack` as a dependency since:
- the dependency is outdated and no longer maintained
- the dependency has high security vulnerabilities
- the dependency does not support newer versions of webpack

The change now uses the already used `css-loader` and `url-loader` to load font-awesome icons and fonts.



#### How to test


1, confirm that the build and start of the application is successful
2. confirm that `yarn audit --level=high` does not yield vulnerability results for `font-awesome-webpack`
3. confirm that the use of `font-awesome` icons still works correctly (ex: replacing a codicon with the previous `fa`)

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
